ci-operator/config/openshift/jobset-operator/openshift-jobset-operator-main.yaml

```diff
@@ -72,7 +72,7 @@ tests:
       commands: make test-e2e
       dependencies:
       - env: RELATED_IMAGE_OPERAND_IMAGE
-        name: kubernetes-sigs-jobset
+        name: pipeline:kubernetes-sigs-jobset
```

Comment:
This is correct. Operand should get this from pipeline as https://github.com/openshift/release/blob/11f36d43c71a38e5778cd05f16869337fb6c6caf/ci-operator/config/openshift/lws-operator/openshift-lws-operator-main.yaml#L77.

Since currently operand is not buildable (due to go 1.24 issue), there is no image in here.